### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Created landing page. Kindly review it and let me know what do you think about it !

### CODE REVIEW

File: Authentication/Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        padding:2rem;
+        background-color: var(--lightgray-color);
+        opacity: 0.9;
+        margin: 2rem;
+    }
+
+
+}
+
+
```
Comment (Ankur9669) [inline Authentication/Login/Login.css:143]: Why login.css in homepage🤔

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
I think you have put all your code in the same branch, so first i would request you to solve this issue.

For now i have only reviewed you UI.


image
1. Adding a little padding beetween the sections would make it look good.

image

1. The image is blurry is this intensional?

image
1. Can add a little padding inside the card.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
I agree with Ankur, you should segregate PRs and resolve this branch issue, as this PR has code for other things as well. When Neogcamp team will look into your project and PRs for assessment, it can confuse them.

I have suggested UI changes that need to be resolved for better user experience.

Screenshot 2022-02-11 142735
Your image is getting blurred because its intrinsic value is less than rendered dimensions.

 
Capture
I believe these will be clickable in the future, so it is semantically better to create them as buttons rather than divs.

Capture2
Text is overflowing and overlapping on other elements.



Capture3
There should be consistency in their sizes and also add some on hover effect and cursor: pointer. Also since cart will be button, it is better to use button tag with Font Awesome icons in it than divs containing img.

  
Screenshot 2022-02-11 144643
Screenshot 2022-02-11 144718
When I hover over card buttons, card height increases because of border getting added to button on hover, that should be fixed as it does not give good user experience.

File: Common.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    line-height:1rem;
+}
+
+.flex-container-row{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    /* background-color: purple; */
+    gap:20px;
+    justify-content: center;
```
Comment (Ankur9669) [inline Common.css:19]: Could remove commented code and could convert the px into rem if you want

File: LandingPage/LandingPage.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
+@import '../Common.css';
+
+.mySlides {
+    display:none;
+}
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:5]: Dont use camel casing in css

File: LandingPage/LandingPage.css [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+/*--Slider--*/
+.slider{
+	width: 100%;
+	/* height: 30%; */
+    height: 80vh;
+}
+.sliderImg img{
+	width: 100%;
+	height: 65vh;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:15]: Same you have used camel casing here

File: LandingPage/LandingPage.css [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+.sliderImg img:hover{
+	color: orange;
+}
+
+.carousel-caption{
+	top: 50%;
+	transform: translateY(50%);
+	bottom: initial!important;
+}
+
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:37]: Why the use of !important?? This could cause issues

File: LandingPage/LandingPage.css [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+    height:max-content;
+    width:100%;
+    gap:1rem;
+}
+
+.step-item{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap; 
+    justify-content: center;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:103]: By default flex-wrap is nowrap only so you can remove it

File: LandingPage/LandingPage.css [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.featured-categories .card-container .card-img{
+    height: 15rem;
+}
+
+.flex-container-row.featured-prdcts{
+    background-color: var(--linen-color);
+    flex-wrap: nowrap;
+    overflow: scroll;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:140]: Same you can remove flex-wrap

File: LandingPage/LandingPage.css [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+
+/*--Section for - Testimonials grid--*/
+.flex-container-row.reviews-container{
+    background-color: var(--tertiary-color);
+}
+
+
+#Testimonials{
+	padding-top: 1rem;
+	padding-bottom: 1rem;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css]: If possible dont use id's and in Testimonials "T" should be small

File: LandingPage/LandingPage.css [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+}
+.flex-container-row.reviews-row{
+    flex-wrap: nowrap;
+    padding-top: 2rem;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css]: Can remove nowrap

File: LandingPage/LandingPage.css [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+}
+.flex-container-row.reviews-row .profile{
+	padding: 2rem 1rem 1rem;
+	background-color: var(--lightgray-color);
+    text-align: center;
+}
+.flex-container-row.reviews-row .user{
+	width:120px;
+	height: 120px;
+	border-radius: 50%;
```
Comment (Ankur9669) [inline LandingPage/LandingPage.css:209]: Could use rems

File: LandingPage/LandingPage.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+function showDivs(n) {
+  var i;
+  var x = document.getElementsByClassName("mySlides");
+  if (n > x.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = x.length}
+  for (i = 0; i < x.length; i++) {
+    x[i].style.display = "none";  
+  }
+  x[slideIndex-1].style.display = "block";  
+}
```
Comment (Ankur9669) [inline LandingPage/LandingPage.js:17]: Could use es6 here

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
Could have more margin, and equal top and bottom margin.


image

This cards container could have more top and bottom padding.

image

1. The card lorem ispum text is very small and difficult to read, so you can increase the font-size.
2. And also there could be some margin beetwen the loremispum and the heading of the card

image

These could have more top and bottom margins.

image

The badge is not properly aligned with the heading text

image

The 299 price is not centered vertically

image

Margin is inconsistent in bottom it is more but on top it is less

File: Common.css (hunk 2 above)
Comment (kgajriaAgarwal) [reply Common.css:19]: removed

File: LandingPage/LandingPage.css (hunk 3 above)
Comment (kgajriaAgarwal) [reply LandingPage/LandingPage.css:5]: removed came casing

File: LandingPage/LandingPage.css (hunk 4 above)
Comment (kgajriaAgarwal) [reply LandingPage/LandingPage.css:15]: removed came casing

**Comment by kgajriaAgarwal** [comment]
done changes acc to the above comments.

**Pull request closed without merging**